src/GafferUI/AnnotationsGadget.cpp

```diff
@@ -200,7 +224,44 @@ void AnnotationsGadget::doRenderLayer( Layer layer, const Style *style ) const
 			const Imath::Color3f textColor( 1.0f );
 			glPushMatrix();
 				IECoreGL::glTranslate( V2f( b.min.x + 1.0 - textBounds.size().x * 0.5, b.max.y - textBounds.size().y * 0.5 - 0.7 ) );
-				style->renderText( Style::LabelText, annotations.numericBookmark.string(), Style::NormalState, &textColor );
+				style->renderText( Style::BodyText, annotations.numericBookmark.string(), Style::NormalState, &textColor );
+			glPopMatrix();
+		}
+
+		if( annotations.standardAnnotations.size() )
+		{
+			glPushMatrix();
+			IECoreGL::glTranslate( V2f( b.max.x + g_offset + g_borderWidth, b.max.y - g_borderWidth ) );
+
+			const Color3f midGrey( 0.65 );
+			const Color3f darkGrey( 0.05 );
+			float previousHeight = 0;
+			for( auto &a : annotations.standardAnnotations )
```

Comment:
> Why is `a` not const here?

Just an oversight - fixed in 4320d1b75.

> I guess we're ok with abbreviating variable names that aren't visible to the public? I think I've never really used abbreviations because I thought you'd prefer the long version :)

I can't say I'm terribly consistent on this, and I do generally prefer things to be spelled out, but I guess that in this case I guess I felt that I could remember what `a` stood for for the duration of that short block. I think I was also struggling with the ambiguity between the Annotations class and the StandardAnnotations that it contained, and felt that `a` was less confusing than `annotation` and `standardAnnotation` was just annoyingly verbose.

> Also, general question: I think initially when we switched to C++ 11 you were a little hesitant to use auto for anything but iterators and stuff like that, is that true? Are you ok with using auto a bit more now?

Yeah, I was hesitant because I don't use fancy IDEs and I like to know what types I'm dealing with. But this old fuddy-duddy is warming to `auto` a bit more. I think my vague strategy is probably to allow it to gradually creep into more of my code until one day it burns me :) I think my tastes probably waver, but there's a few places I'm totally sold on it, in addition to the iterators. I see it as a definite improvement where the type is already obvious from the rest of the line for instance, like `auto m = runTimeCast<MeshPrimitive>( o )` or `auto p = make_shared<X>()`.

Reply:
Cool, thanks for the additional explanations :) I'll try to follow those preferences.